### PR TITLE
Relay runtime: add "lazy" and "debug" logger

### PR DESCRIPTION
**Pull request by mrtnzlml**
I've seen something similar in one FB talk which make me realize that we are logging unnecessary garbage. So I renamed the current logger to "eager" logger and created a new "lazy" one. The difference is, that this logger waits for the response and prints all the information at once (instead of streaming them into console as they arrive). This is a first iteration but eventually I'd like to use the lazy logger as a default one inside our Relay package.

### CODE REVIEW

File: src/relay-runtime/src/__tests__/RelayLazyLogger.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/**
+ * @flow
+ * @jest-environment jsdom
```
Comment (tbergq) [inline src/relay-runtime/src/__tests__/RelayLazyLogger.test.js:3]: Isn't this already what we use in this workspace?

File: src/relay-runtime/README.md [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
+An opinionated Relay Runtime wrapper with nice query logging.
 
 ## Install
 
 `yarn add @adeira/relay-runtime`
 
 ## Usage
 
-### Logger
+TKTK
```
[the remark this answers is not in the document]
Comment (mrtnzlml) [reply src/relay-runtime/README.md:9]: yes, would be nice to write some docs but I don't want to do it now 😬

**Review by tbergq — APPROVED** [review]

File: src/relay-runtime/src/__tests__/RelayLazyLogger.test.js (hunk 1 above)
Comment (mrtnzlml) [reply src/relay-runtime/src/__tests__/RelayLazyLogger.test.js:3]: It is, however, we should also have a Node.js test which is currently missing. You are the author actually in the eager logger and I like to keep it there explicitly. 😊

Comment (tbergq) [reply src/relay-runtime/src/__tests__/RelayLazyLogger.test.js:3]: Sure, makes sense 👌

**Pull request merged**